pkg/operation/botanist/monitoring.go

```diff
@@ -635,8 +619,13 @@ func (b *Botanist) deployGrafanaCharts(ctx context.Context, credentialsSecret *c
 	)
 }
 
-// DeleteGrafana will delete all grafana instances from the seed cluster.
+// DeleteGrafana will delete all grafana resources from the seed cluster.
 func (b *Botanist) DeleteGrafana(ctx context.Context) error {
+	return common.DeleteGrafana(ctx, b.SeedClientSet, b.Shoot.SeedNamespace)
+}
+
+// DeleteDeprecatedGrafana will delete all deprecated grafana instances from the seed cluster.
+func (b *Botanist) DeleteDeprecatedGrafana(ctx context.Context) error {
 	if err := common.DeleteGrafanaByRole(ctx, b.SeedClientSet, b.Shoot.SeedNamespace, grafanaOperatorsRole); err != nil {
```

Comment:
Maybe we could mark the functions with a TODO "to be removed" that are not needed after the migration.